.github/workflows/build.yml

```diff
@@ -58,17 +59,24 @@ jobs:
          cd $GITHUB_WORKSPACE
          mkdir build
          cd build
-         cmake -DCMAKE_TOOLCHAIN_FILE="${{ env.TOOLCHAIN_FILE }}" -DRDK_PLATFORM=DEV_VM -DCMAKE_INSTALL_PREFIX:PATH=/usr -DENABLE_DOBBYL1TEST=ON -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} ${{ matrix.extra_flags }} ${{ env.optional_plugins }} ..
+         if [ $matrix.extra_flags = "RUN_TESTS" ]
+         then
+             cmake -DCMAKE_TOOLCHAIN_FILE="${{ env.TOOLCHAIN_FILE }}" -DRDK_PLATFORM=DEV_VM -DCMAKE_INSTALL_PREFIX:PATH=/usr -DENABLE_DOBBYL1TEST=ON -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} ${{ matrix.optional_flags }} ${{ env.optional_plugins }} ..
+         else
+             cmake -DCMAKE_TOOLCHAIN_FILE="${{ env.TOOLCHAIN_FILE }}" -DRDK_PLATFORM=DEV_VM -DCMAKE_INSTALL_PREFIX:PATH=/usr -DENABLE_DOBBYL1TEST=ON -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} ${{ matrix.extra_flags }} ${{ env.optional_plugins }} ..
```

Comment:
Why do we need to enable L1 test build option (ENABLE_DOBBYL1TEST)  if we are not going to run it ?

Reply:
Yes, Not required to enable ENABLE_DOBBYL1TEST option if we are not running the test. Now I have removed it. thanks